lms/djangoapps/certificates/models.py

```diff
@@ -122,10 +122,8 @@ class GeneratedCertificate(models.Model):
     status = models.CharField(max_length=32, default='unavailable')
     mode = models.CharField(max_length=32, choices=MODES, default=MODES.honor)
     name = models.CharField(blank=True, max_length=255)
-    created_date = models.DateTimeField(
-        auto_now_add=True, default=datetime.now)
-    modified_date = models.DateTimeField(
-        auto_now=True, default=datetime.now)
+    created_date = models.DateTimeField(auto_now_add=True)
+    modified_date = models.DateTimeField(auto_now=True)
```

Comment:
Default is not needed: https://docs.djangoproject.com/en/1.7/ref/models/fields/#datefield.